lib/pages/chat/chat.dart

```diff
@@ -404,16 +405,16 @@ class ChatController extends State<Chat> with ImagePickerMixin, SendFilesMixin {
         break;
       case PermissionStatus.permanentlyDenied:
         showDialog(
-          useRootNavigator: false,
-          context: context,
-          builder: (context) {
-            return PermissionDialog(
-              permission: Permission.storage,
-              explainTextRequestPermission: Text(L10n.of(context)!.explainGoToStorageSetting),
-              icon: const Icon(Icons.preview_outlined),
-            );
-          }
-        );
+            useRootNavigator: false,
+            context: context,
+            builder: (context) {
+              return PermissionDialog(
+                permission: Permission.storage,
+                explainTextRequestPermission:
+                    Text(L10n.of(context)!.explainGoToStorageSetting),
+                icon: const Icon(Icons.preview_outlined),
+              );
+            });
         break;
       case PermissionStatus.restricted:
```

Comment:
IMO, should add `default: break` for safety, because if new permission is created and we have not handled may create unexpect behavior

Reply:
fixed